folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -113,6 +118,34 @@ public static CompletionStage<RawMemcacheClient> connect(
       final int maxSetLength,
       final EventLoopGroup eventLoopGroup,
       final Class<? extends Channel> channelClass) {
+    return connect(
+        address,
+        outstandingRequestLimit,
+        eventLoopThreadFlushMaxBatchSize,
+        binary,
+        executor,
+        connectionTimeoutMillis,
+        charset,
+        metrics,
+        maxSetLength,
+        eventLoopGroup,
+        channelClass,
+        null);
+  }
+
+  public static CompletionStage<RawMemcacheClient> connect(
```

Comment:
Do we need the extra constructor here? This is not part of the public API so would be fine to change the existing one.

Reply:
Makes sense, fixed